src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -160,8 +152,6 @@ private bool WaitForExitCore(int milliseconds)
                 {
                     _error.WaitUtilEOF();
                 }
-
-                ReleaseProcessHandle(handle);
```

Comment:
Is there a reason we were previously closing the handle at the end?  Seems purposeful, but maybe not.  I'm just wondering if by disposing of it earlier now, e.g. before these output/error waits, if we're potentially messing something up.